mitodl/mitxpro
# profile add highest level of education

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #831 

#### What's this PR do?
Add highest level of education in profile.

#### How should this be manually tested?

- Create new account
- You should see new field `Highest Level of Education`.
- You should also see above field in profile detail and on edit page.

#### Screenshots (if appropriate)
**On create/edit form**
<img width="886" alt="Screenshot 2019-08-05 at 6 11 18 PM" src="https://user-images.githubusercontent.com/4245618/62467151-76fc1280-b7ac-11e9-8215-877c13c9df11.png">

**On detail page**
<img width="888" alt="Screenshot 2019-08-05 at 6 09 39 PM" src="https://user-images.githubusercontent.com/4245618/62467067-559b2680-b7ac-11e9-99bc-542062eb0aff.png">
